dependencies/sigil-pypi-dependencies.json

```diff
@@ -0,0 +1,134 @@
+{
+    "name": "sigil-pypi-dependencies",
+    "buildsystem": "simple",
+    "build-commands": [],
+    "modules": [
+        {
+            "name": "python3-lxml",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"lxml\" --no-build-isolation"
```

Comment:
you need to add a `--ignore-installed`

for some reason python3-xml is in the SDK but not the platform.

Reply:
Seems with `--exists-action=i --prefix=${FLATPAK_DEST}`, it already ignores installed.